CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 
 #### Features
 
+* [#1308](https://github.com/ruby-grape/grape/pull/1308): Add admin only custom validation example to doc and spec - [@ashkan18](https://github.com/ashkan18)
```

Comment:
Missing a period at the end. But we don't really need a CHANGELOG line for docs + spec really, though we absolutely do appreciate the contribution!